src/auslib/web/public/client.py

```diff
@@ -36,11 +36,18 @@ def getSystemCapabilities(systemCapabilities):
     # New-style SYSTEM_CAPABILITIES, as implemented in https://bugzilla.mozilla.org/show_bug.cgi?id=1373367
     if systemCapabilities.startswith("ISET:"):
         for part in systemCapabilities.split(","):
+            # Skip fields with an unparseable format, which we see often. Eg:
+            # ISET:SSE4_2,MEM:32768,(select*from(select(sleep(20)))a)
+            if ":" not in part:
+                continue
             key, value = part.split(":", 1)
             if key == "ISET":
                 caps["instructionSet"] = value
             elif key == "MEM":
-                caps["memory"] = int(value)
+                try:
+                    caps["memory"] = int(value)
+                except ValueError:
```

Comment:
`int()` can also raise `TypeError`, but probably not in this case, where use already ensure that the value passed is a string.

Reply:
TIL! Let's address this if it comes up.